components/gitpod-protocol/src/context-url.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+export namespace ContextURL {
+  export const INCREMENTAL_PREBUILD_PREFIX = "incremental-prebuild";
```

Comment:
Could you add at least the `imagebuild/` here as well? Don't know whether `additionalcontent` makes also sense (but that's a regex).

![image](https://user-images.githubusercontent.com/24960040/123651052-726ef600-d82b-11eb-9258-571947c9df5f.png)